test/BoletoTest.java

```diff
@@ -0,0 +1,43 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BoletoTest {
+        @Test
+        public void testDescuentos(){
+            Cliente cliente = new Cliente(10101001, "DIEGO FIGUEROA", "Boliviano", "03/01/2001");
+            Sala sala = new Sala("5");
+            Boleto boleto = new Boleto(cliente, 40, sala);
+
+            //1
+            boleto.aplicarDescuento(Boleteria.MetodoPago.TARJETA,Semana.JUEVES);
+            assertEquals(35.2,boleto.precioPelicula);
+            assertEquals(44,cliente.getPuntosTotales());
+
+
+            //2
+            Cliente cliente1 = new Cliente(10101001, "DIEGO FIGUEROA", "Boliviano", "03/01/2018");
+            Sala sala1 = new Sala("6");
+            Boleto boleto1 = new Boleto(cliente1, 60, sala1);
+            boleto1.setGeneroPelicula("Animacion");
+
+            boleto1.aplicarDescuento(Boleteria.MetodoPago.EFECTIVO,Semana.DOMINGO);
+            assertEquals(51,boleto1.precioPelicula);
+            assertEquals(42,cliente1.getPuntosTotales());
+
+
+            //3 CHOQUE UN MENOR DE EDAD QUE VE UNA ANIMACION UN DIA MIERCOLES
```

Comment:
Todo esta bien, podrias separar los tipos de test con funciones dentro de tu test, en lugar de separarlos mediante comentarios.
```suggestion
        @Test
        public void testChoqueDeDescuentos(){
        }
```